### PR TITLE
[Update 8] Update release note with LS changes

### PR DESCRIPTION
**Pull request by KavinduZoysa**
## Purpose
$subject

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -90,14 +90,28 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
 
 ### New features
 
+#### Language Server
+
+- Add custom completions for http services
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:95]:
```suggestion
- Add custom completions for HTTP services.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -90,14 +90,28 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
...
+
 #### CLI
 
 - Added a new `bal tool` command to manage tools that extend the functionality of the CLI. Tools can be pulled from the Ballerina Central and are managed using the `bal tool` command. For more information, see [Tool commands](https://ballerina.io/learn/cli-documentation/cli-commands/#tool-commands).
 
 ### Improvements
 
+#### Language Server
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:103]:
```suggestion
#### Language Server
```
Add full stops for all the bullet points.

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 3]
```diff
@@ -90,14 +90,28 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
...
+
+- Suggest ballerina central packages when they're partially typed
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:105]:
```suggestion
- Suggest Ballerina Central packages when they are partially typed
```

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 4]
```diff
@@ -90,14 +90,28 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
...
+- Improve sorting in Record type descriptor node context
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:106]:
```suggestion
- Improve sorting in the Record-type descriptor node context.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 5]
```diff
@@ -90,14 +90,28 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
...
+- Introduce a codeaction to add local module dependencies to Ballerina.toml
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:107]:
```suggestion
- Introduce a code action to add local module dependencies to the `Ballerina.toml` file.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 6]
```diff
@@ -90,14 +90,28 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
...
+- Introduce change variable type code action for let expression
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:108]:
```suggestion
- Introduce a code action to change the variable type of a `let` expression.
```
Be consistent with the above format.

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 7]
```diff
@@ -90,14 +90,28 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
...
+- Introduce create function code action for the expression of select clause
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:109]:
```suggestion
- Introduce a code action to create a function for the expression of a `select` clause.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 8]
```diff
@@ -90,14 +90,28 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
...
+- Improve completions in Service Declaration node context
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:110]:
```suggestion
- Improve completions in the service declaration node context.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 9]
```diff
@@ -90,14 +90,28 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
...
+- LS simulator improvements
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:111]:
```suggestion
- Improved the LS simulator.
```
Need to specify the improvements precisely.

**Review by mindula — APPROVED** [review]

**Pull request merged**